source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,135 @@
+.. _php-cluster-monitoring:
+
+==================
+Cluster Monitoring
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, server, topology
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to use the {+php-library+} to monitor topology
+events in a MongoDB instance, replica set, or sharded cluster. The
+driver creates topology events, also known as Server Discovery and
+Monitoring (**SDAM**) events, when there are any changes in the state of the
```

Comment:
S: i know SDAM is the more commonly used term, but i wonder if bolding the whole thing makes more sense

```suggestion
driver creates topology events, also known as **Server Discovery and
Monitoring (SDAM)** events, when there are any changes in the state of the
```

Reply:
probably ok to go without any bolding, tbh